app/controllers/api/v1/restaurants_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::RestaurantsController < ApplicationController
+
+  def index
+    @restaurants = Restaurant.all
```

Comment:
I would try `@restaurants = Restaurant.all.includes(:restaurant_categories)` to save some db calls and avoide setting the `@categories` variable`